pkg/api/types.go

```diff
@@ -543,6 +543,31 @@ type PersistentVolumeClaimSpec struct {
 	StorageClassName *string
 }
 
+type PvcConditionType string
+
+// These are valid conditions of Pvc
+const (
+	// An user trigger resize of pvc has been started
+	PvcResizeStarted PvcConditionType = "ResizeStarted"
```

Comment:
api-conventions.md says: *Do not use abbreviations in the API, except where they are extremely commonly used, such as "id", "args", or "stdin".*

I am afraid that full PersistentVolumeClaimResizeStarted is necessary (+all other PVC abbreviations in this file should be expanded).

Reply:
Initialisms in Go type fields must be capitalized `PVCResizeStarted` not `PvcResizeStarted`.  Another example: `urlField` and `URLField` are correct, not `UrlField`.

Reply:
But the general comment holds, please expand all of these to be `PersistentVolumeClaimXXX`